sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,108 @@
+FROM golang:1.11
+
+ADD . /firecracker-containerd/
+
+ARG TMP_BUILD_DIR=/tmp/build
+ARG FIRECRACKER_SRC_DIR="/firecracker"
+ARG FIRECRACKER_BUILD_DIR="$FIRECRACKER_SRC_DIR/build"
+ARG CARGO_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_registry"
+ARG FIRECRACKER_CONTAINERD_SRC_DIR="/firecracker-containerd"
+ARG FIRECRACKER_VERSION="0.15.0"
+
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV PATH="/go/bin:$CARGO_HOME/bin:$PATH"
+ENV CARGO_TARGET_DIR="$FIRECRACKER_BUILD_DIR/cargo_target"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        binutils-dev \
+        cmake \
+        curl \
+        file \
+        g++ \
+        gcc \
+        git \
+        iperf3 \
+        iproute2 \
+        jq \
+        libcurl4-openssl-dev \
+        libdw-dev \
+        libiberty-dev \
+        libssl-dev \
+        lsof \
+        make \
+        musl-tools \
+        net-tools \
+        openssh-client \
+        pkgconf \
+        zlib1g-dev \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
+	sudo \
+	iptables \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the Rust toolchain
+#
+# Install 1.32.0 due to Firecracker crashing with 148 code
+RUN mkdir "$TMP_BUILD_DIR" \
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y \
```

Comment:
I'd like to see this done the same way as @sipsma is doing in https://github.com/firecracker-microvm/firecracker-containerd/pull/134/files#diff-6e72c6a06c3291e7fccf443ca341959cR70